e2e/cypress/integration/cms_login_spec.js

```diff
@@ -0,0 +1,29 @@
+// CMS Login Test
+
+describe('CMS Login Test', () => {
+    
+    beforeEach(() => {
+        cy.visit('http://localhost:8082/');
```

Comment:
If you use the project network:

```suggestion
        cy.visit('http://edx_cms:8000/');
```

Reply:
This URL is not found, hence I have to use localhost

Reply:
Now it should be if you remove the `network_mode: host` in the docker-compose configuration.